api/v1alpha1/kubeadmbootstrapconfig_types.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// KubeadmBootstrapConfigSpec defines the desired state of KubeadmBootstrapConfig
+type KubeadmBootstrapConfigSpec struct {
+	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+	// Important: Run "make" to regenerate code after modifying this file
+}
+
+// KubeadmBootstrapConfigStatus defines the observed state of KubeadmBootstrapConfig
+type KubeadmBootstrapConfigStatus struct {
+	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
```

Comment:
```suggestion
	// INSERT ADDITIONAL STATUS FIELD - define observed state of KubeadmBootstrapConfig
```

Reply:
This is a generated file and also the beginning of scaffolding. Gonna leave this.